### PR TITLE
production-install: Update instructions for creating initial user.

### PR DESCRIPTION
**Pull request by drewmoseley**
Issues: MEN-1347

Changelog: title

Signed-off-by: Drew Moseley <drew.moseley@northern.tech>

### CODE REVIEW

**Comment by drewmoseley** [comment]
Does this look like the right fix for MEN-1347?  I based this on the change already included in the "Create a Test Environment" section

**Comment by bboozzoo** [comment]
@mchalski want to take a look as well?

File: 06.Administration/02.Production-installation/docs.md [hunk 1]
```diff
@@ -597,12 +597,12 @@ menderproduction_storage-proxy_1              /usr/local/openresty/bin/o ...   U
 <!--AUTOMATION: test=docker ps | grep menderproduction | grep "0.0.0.0:443" -->
 <!--AUTOMATION: test=docker ps | grep menderproduction | grep "0.0.0.0:9000" -->
 
-Furthermore, since this is a brand new installation it should be possible to request initial
-user login token through the API:
+Furthermore, since this is a brand new installation it should be possible to create the initial
+user via the CLI provided by  the User Administration Service. The service's binary is embedded in a Docker container, so to execute it you will issue the **exec** subcommand of docker-compose, e.g.:
 
 <!--AUTOMATION: ignore -->
```
Comment (bboozzoo) [inline 06.Administration/02.Production-installation/docs.md]: @GregorioDiStefano do you need to update any of the AUTOMATION tags in this document?

**Comment by drewmoseley** [comment]
@GregorioDiStefano @mchalski is this important for the 1.2 release?

File: 06.Administration/02.Production-installation/docs.md (hunk 1 above)
Comment (GregorioDiStefano) [inline 06.Administration/02.Production-installation/docs.md]: Hmm, let me look into this.

**Comment by GregorioDiStefano** [comment]
@drewmoseley yes, it is. This is the new procedure requires for 1.2.x

**Comment by drewmoseley** [comment]
@GregorioDiStefano can you approve the review?
@mchalski?

**Comment by drewmoseley** [comment]
Also I added a commit here to fix a typo.

**Comment by GregorioDiStefano** [comment]
@drewmoseley let me try adding the useradm bit today to this pr

**Comment by drewmoseley** [comment]
@GregorioDiStefano I'm confused.  What useradm bit are you referring to?

**Comment by GregorioDiStefano** [comment]
@drewmoseley I was going to add an "AUTOMATION: " tag that create the initial user and logs in, but, I realized there is little benefit in doing so, as this is already covered in integration tests. So ignore my comment :)

**Pull request merged**